### PR TITLE
chore: update wif

### PR DESCRIPTION
**Pull request by verbanicm**

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]
Looks like the new Go linter is unhappy

**Comment by verbanicm** [comment]
@sethvargo uh yes, this is gonna be rough

**Comment by sethvargo** [comment]
@verbanicm it should just be changing to "getters"

**Review by sethvargo — APPROVED** [review]

**Review by sethvargo — APPROVED** [review]

File: clients/go/test/grpc-app/main.go [hunk 1]
```diff
@@ -172,17 +172,17 @@ func (s *server) Addition(svr talkerpb.Talker_AdditionServer) error {
...
+		sum += int(req.GetAddend())
 	}
 
 	return nil
 }
 
 func (s *server) Fail(ctx context.Context, req *talkerpb.FailRequest) (*talkerpb.FailResponse, error) {
 	if logReq, ok := audit.LogReqFromCtx(ctx); ok {
-		logReq.Payload.ResourceName = req.Target
+		logReq.Payload.ResourceName = req.GetMessage()
```
Comment (yolocs) [inline clients/go/test/grpc-app/main.go:185]:
```suggestion
		logReq.Payload.ResourceName = req.GetTarget()
```

**Review by yolocs — APPROVED** [review]

**Pull request merged**